src/pages/Calculator.tsx

```diff
@@ -146,6 +153,13 @@ export const Calculator = (): React.ReactElement => {
     calculatorData.interaction,
   )
 
+  const editScenario =
```

Comment:
nit: could this be scenarioSelected or scenarioEditable?

Reply:
done